### ISSUE TITLE
Don't forget about Iceland

### ISSUE THREAD

**Issue by brynpickering** [timtroendle/euro-calliope#15]
Iceland is always left out of European energy system models, poor thing.

Most European datasets already cover Iceland, and a DC interconnector is expected between it (her/him/them?) and the UK, so it seems worthwhile. Only thing is that we'll need to update some upstream dependencies, such as the datasets of technically eligible land area, currently grabbed from a [static Zenodo repository](https://zenodo.org/record/3533038).

**Comment by timtroendle** [comment]
Good idea. If the input data sets of [the renewable potentials source](https://github.com/timtroendle/possibility-for-electricity-autarky) all cover Iceland, determining the technically eligible land area and updating the Zenodo repository will be easy.
